packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -3511,15 +3496,13 @@ describe('packages/date-picker', () => {
           userEvent.tab();
           errorElement = queryByTestId('lg-form_field-error_message');
           expect(errorElement).toHaveTextContent(
-            '2020-02-30 is not a valid date',
+            '2020-02- is not a valid date',
           );
 
           userEvent.type(dayInput, '{backspace}{backspace}');
           userEvent.tab();
           errorElement = queryByTestId('lg-form_field-error_message');
-          expect(errorElement).toHaveTextContent(
-            '2020-02- is not a valid date',
-          );
+          expect(errorElement).toHaveTextContent('2020-- is not a valid date');
```

Comment:
Not for this PR, but we should improve this error message

Reply:
I can create a ticket for this

Reply:
https://jira.mongodb.org/browse/LG-4028